clients/java/src/main/java/io/odpf/stencil/client/URLStencilClient.java

```diff
@@ -38,17 +38,17 @@ public URLStencilClient(String url, StencilConfig config, DescriptorCacheLoader
     }
 
     public URLStencilClient(String url, StencilConfig stencilConfig, DescriptorCacheLoader cacheLoader, Ticker ticker) {
-        this.shouldAutoRefreshCache = stencilConfig.shouldAutoRefreshCache();
-        this.ttl = stencilConfig.getTilInMinutes() != 0 ? Duration.ofMinutes(stencilConfig.getTilInMinutes()) :
+        this.shouldAutoRefreshCache = stencilConfig.getCacheAutoRefresh();
+        this.ttl = stencilConfig.getCacheTtlMs() != 0 ? Duration.ofMillis(stencilConfig.getCacheTtlMs()) :
```

Comment:
The conversion to Duration, I find it unnecessary, as we are using long everywhere